src/simplewallet/simplewallet.cpp

```diff
@@ -1462,6 +1464,72 @@ bool simple_wallet::handle_command_line(const boost::program_options::variables_
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::prompt_for_new_node()
+{
+    bool connect = false;
+    std::string node_url;
+
+    message_writer() << tr("Do you want to connect to a remote node?");
```

Comment:
Same about wording.

Reply:
I think in this case it is good to emphasize that a node from the `https://www.xmr.be/` list would be a remote node. If no is answered a localhost node can be enabled.

Reply:
Sure but you can enter a local IP. I guess it doesn't really matter.